shared/src/msi-installer/WindowsInstaller.wixproj

```diff
@@ -22,7 +22,7 @@
     <MonitoringHomeDirectory Condition="'$(MonitoringHomeDirectory)' == ''">$(MSBuildThisFileDirectory)..\..\bin\monitoring-home</MonitoringHomeDirectory>
     <TracerHomeDirectory Condition="'$(TracerHomeDirectory)' == ''">$(MSBuildThisFileDirectory)..\bin\windows-tracer-home</TracerHomeDirectory>
     <ProfilerHomeDirectory Condition="'$(ProfilerHomeDirectory)' == ''">$(MSBuildThisFileDirectory)..\bin\DDProf-Deploy</ProfilerHomeDirectory>
-    <DefineConstants>InstallerVersion=1.28.6;MonitoringHomeDirectory=$(MonitoringHomeDirectory);TracerHomeDirectory=$(TracerHomeDirectory);LibDdwafDirectory=$(LibDdwafDirectory);ProfilerHomeDirectory=$(ProfilerHomeDirectory)</DefineConstants>
+    <DefineConstants>InstallerVersion=1.31.0;MonitoringHomeDirectory=$(MonitoringHomeDirectory);TracerHomeDirectory=$(TracerHomeDirectory);LibDdwafDirectory=$(LibDdwafDirectory);ProfilerHomeDirectory=$(ProfilerHomeDirectory)</DefineConstants>
```

Comment:
The last tracer version is 2.0.0-prerelease. Why aren't you taking this one out of curiosity?

Reply:
Also, It won't let me comment but shouldn't you change line 20 as well (OutputName)?

Reply:
> The last tracer version is 2.0.0-prerelease. Why aren't you taking this one out of curiosity?
We have alpha/beta customer who are running .NET 4.5.X.

> Also, It won't let me comment but shouldn't you change line 20 as well (OutputName)?
I'll have a look

Reply:
> Shouldn't we update the version bump workflow instead of doing that manually?

I look into it too :)

Reply:
After looking around, I might do this in another PR if it's ok. WDYT?

Reply:
Sure makes sense. I actually even don't know how we handle new versions for a branch. And here, it's even more special, as you want to stick with the 1.x branch, on master. So maybe manual is the way to go for now? 🤷‍♂️